kiwi_lint/field_label.py

```diff
@@ -0,0 +1,27 @@
+from pylint import checkers
+from pylint import interfaces
+
+
+class FormFieldLabelChecker(checkers.BaseChecker):
+    """
+    Warns against defining labels in Form Field class constructors.
+    Labels must be placed in the HTML.
+    """
+    __implements__ = (interfaces.IAstroidChecker, )
+
+    name = 'form-field-label-checker'
+
+    msgs = {
+        'R4811': ('Do not use label in form Field class constructor.',
+                  'form-field-label-used',
+                  'Do not use label in form Field class constructor,'
+                  'place template label in HTML. See:'
+                  'https://github.com/kiwitcms/Kiwi/issues/738')
+    }
+
+    def visit_call(self, node):
+        if node.func.as_string().endswith('Field') and node.keywords:
```

Comment:
Can also do func.attrname

Reply:
Looking at the [docs](https://greentreesnakes.readthedocs.io/en/latest/nodes.html#Call) the func here can either be a Name or an Attribute object. `func.attrname` works for Attribute objects but not for Name objects, which store the name in `func.name`. I think leaving it at `.as_string` would be best to account for both.

Reply:
In this specific case the underlying node type is Attribute b/c we call is like `forms.<FieldClass>`

I think if that was only `FieldClass()` then it would be a Name type node:

```
>>> m = astroid.parse("""name = forms.CharField(required=True, label='missing')""")
>>> print(m.repr_tree())
Module(
   name='',
   doc=None,
   file='<?>',
   path=['<?>'],
   package=False,
   pure_python=True,
   future_imports=set(),
   body=[Assign(
         targets=[AssignName(name='name')],
         value=Call(
            func=Attribute(
               attrname='CharField',
               expr=Name(name='forms')),
            args=[],
            keywords=[Keyword(
                  arg='required',
                  value=Const(value=True)),
               Keyword(
                  arg='label',
                  value=Const(value='missing'))]))])
```

but I accept the argument so let's leave it like that.